app/src/behavior.c

```diff
@@ -17,11 +17,18 @@
 
 #endif
 
+#include <zmk/ble.h>
+#if ZMK_BLE_IS_CENTRAL
```

Comment:
```suggestion
#if ZMK_BLE_IS_CENTRAL
#include <zmk/ble.h>
```

Reply:
This one is actually to get `ZMK_BLE_IS_CENTRAL` itself (and I believe nothing else, actually).

Reply:
Ah, then disregard. Thanks.